src/compiler/diagnosticMessages.json

```diff
@@ -4954,5 +4954,9 @@
     "No value exists in scope for the shorthand property '{0}'. Either declare one or provide an initializer." :{
         "category": "Error",
         "code": 18004
+    },
+    "Quoted constructors have previously been interpreted as methods, which is incorrect. In TypeScript 3.6, they will be correctly parsed as constructors. In the meantime, consider using 'constructor()' to write a constructor, or '[\"constructor\"]()' to write a method.": {
+        "category": "Error",
+        "code": 96000

```

Comment:
We can use a normal code here; it's fine to introduce a gap in the sequence later

Reply:
I was confused about why `18004` was way down here. Now I see that's the latest of an earlier sequence.

Reply:
Updated to 18005